docs/maintainers/README.md

```diff
@@ -28,6 +28,106 @@ The final step to preparing your language to accept contributions is to provide
 
 We recommend choosing a Concept that involves working with basic types, like strings and numbers. Having chosen a Concept, look for the Concept's description in this reference part of this repository. This file should have references to all tracks that have implemented the exercise. You can use these existing implementations as a starting point for your first concept exercise, which you can then tailor to your specific language.
 
+### Planning the track structure
+
+You may find it helpful to create a graph that shows the unlock progression of the exercises for planning which exercises you need, which prerequisites they have and which exercises are closely related to each other.
+
+The website supports flowcharts rendered by [mermaid](https://mermaid-js.github.io/mermaid/#/flowchart). Every code block with the language set to `mermaid` will be rendered as a graph:
+
+    ```mermaid
+        graph LR
+            basic-numbers --> advanced-numbers
+    ```
+
+```mermaid
+    graph LR
+        basic-numbers --> advanced-numbers
+```
+
+Please refer to the [mermaid documentation](https://mermaid-js.github.io/mermaid/#/flowchart) for detailed instructions. Below you can find a few examples of basic mermaid syntax to get you started. The Julia track's [progression.md](./../../languages/julia/reference/progression.md) can also be used as a reference.
```

Comment:
I'm wondering if we need the examples below, or if the reference to the Julia progression is enough. I was feeling slightly overwhelmed.